src/index.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-function wrap(schema, target, hooks = []) {
+function wrap(schema,params, hooks = []) {
+  let {options={},target=params} = params;
```

Comment:
Please make your code consistent with existing code (add space after `,` and `:`, around `=`).